Assets/Scripts/Gameplay/Bullets/BulletController.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using UnityEngine;
+
+namespace Gameplay.Bullets
+{
+    public class BulletController : MonoBehaviour
+    {
+        [SerializeField] private BulletSO bullet;
+        [SerializeField] private float timeToDieOutScreen = 0.5f;
```

Comment:
We may replace this time with a collider which follows the camera, as suggested in Angelo's issue for the basic enemy:
A child object for the camera may have a collider half screen-size away from the camera edge on each side, and a script to delete anything that touches it.